odl/tomo/operators/ray_trafo.py

```diff
@@ -396,41 +397,10 @@ def __init__(self, domain, geometry, **kwargs):
             variant='forward', **kwargs)
 
     def _call_real(self, x_real, out_real, **kwargs):
```

Comment:
I think `_call_real` and `_call_complex` should also go into the implementation. That gives opportunity for optimization.

Reply:
This I think is a very good point. I did this, but I'm not entirely sure that I picked the best solution yet. Since the `_call` is quite a big function I didn't want to duplicate it throughout the implementations. I added a `complexify()` function for now, let me know what you think.